test/support/github/test_helpers.ex

```diff
@@ -33,4 +33,16 @@ defmodule CodeCorps.GitHub.TestHelpers do
       Application.put_env(:code_corps, :github, old_mock)
     end
   end
+
+  @spec with_real_api(do: function) :: any
+  defmacro with_real_api(do: block) do
+    quote do
+      old_mock = Application.get_env(:code_corps, :github)
+      Application.put_env(:code_corps, :github, CodeCorps.GitHub.API)
+
+      unquote(block)
+
+      Application.put_env(:code_corps, :github, old_mock)
+    end
+  end
```

Comment:
This uses the real API to test. I feel fairly confident that we should introduce _some_ acceptance tests rather than letting our production server serve as that fallback. They should be kept to a bare minimum (like in this case testing a sync of a repository should only be done maybe when merging from staging to master).

I'm not comfortable right now, though, with the best way to do this with `tags`. Could use thoughts.

Reply:
Not sure what the perfect approach is, but  we could make a command alias, for example `mix test.acceptance`, which basically calls `mix test --include :acceptance`

Then we setup  `mix test` to auto-exclude tests with the tag `:acceptance`. 

If we want to use the same test in acceptance and mock API mode, that's probably also doable, but we'll have to be creative. I'm not sure what the approach would be in that case.